rfcs/0045-deprecate-url-syntax.md

```diff
@@ -0,0 +1,74 @@
+---
+feature: deprecate_url_syntax
+start-date: 2019-04-28
+author: Michael Raskin
+co-authors: 
+shepherd-leader: Eelco Dolstra
+shepherd-team: Eelco Dolstra, Jonas Pfenniger, Silvan Mosberger
```

Comment:
@zimbatm is your name correct here?

Reply:
no it's "zimbatm" :)

Reply:
@domenkozar or @7c6f434c would you care to fix this?

Reply:
Ah right, sorry.